CondTools/DT/test/validate/DTTPGParametersValidateDBRead.cc

```diff
@@ -93,8 +93,8 @@ void DTTPGParametersValidateDBRead::analyze( const edm::Event& e,
                        sta,
                        sec,
                        nClock, tPhase, DTTimeUnits::counts );
-    if ( ( fabs( ckclock - nClock ) > 0.0001 ) ||
-         ( fabs( ckphase - tPhase ) > 0.0001 ) )
+    if ( ( std::abs( ckclock - nClock ) > 0.0001 ) ||
```

Comment:
this looks like it should just be if ckclock!=nClock
-as changed, I think we will just change one warning for another